corehq/apps/style/static/app_manager/less/new_appmanager/content.less

```diff
@@ -21,6 +21,13 @@
   &.formdesigner-content-wrapper {
     padding: 0;
   }
+  #formdesigner {
+    background-color: @cc-brand-low !important;
+    .fd-content-right .fd-head,
+    .fd-content-right .fd-form-actions {
+      margin-right: -2px;
```

Comment:
sometimes these type of css styles confuse me when i'm looking to make a change because i'm not sure what they are trying to achieve (usually it's subtle). maybe a comment here?